src/lib/config.ts

```diff
@@ -439,6 +439,13 @@ export class default_config {
         // "youtube.com": "google",
     })
 
+    /**
+     * Strict mode will always ensure a domain is open in the correct container, replacing the current tab if necessary.
+     *
+     * Relaxed mode is less aggressive and instead treats container domains as a default when opening a new tab.
+     */
+    autocontainMode: "strict" | "relaxed" = "strict"
```

Comment:
```suggestion
    autocontainmode: "strict" | "relaxed" = "strict"
```

All other settings are lowercase :)